fvm/evm/emulator/state/stateDB.go

```diff
@@ -170,7 +191,11 @@ func (db *StateDB) GetNonce(addr gethCommon.Address) uint64 {
 }
 
 // SetNonce sets the nonce value for the given address
-func (db *StateDB) SetNonce(addr gethCommon.Address, nonce uint64) {
+func (db *StateDB) SetNonce(
+	addr gethCommon.Address,
+	nonce uint64,
+	reason gethTracing.NonceChangeReason,
```

Comment:
For some reason, the Geth folks introduced this `gethTracing.NonceChangeReason` function parameter, but it's not used anywhere within the method's body.
See: https://github.com/ethereum/go-ethereum/blob/v1.15.8/core/state/statedb.go#L435-L440